File: decompiled/sources/com/hjq/toast/config/IToast.java
package com.hjq.toast.config;

import android.view.View;
import android.widget.TextView;

public interface IToast {
    void cancel();

    TextView findMessageView(View view);

    int getDuration();

    int getGravity();

    float getHorizontalMargin();

    float getVerticalMargin();

    View getView();

    int getXOffset();

    int getYOffset();

    void setDuration(int i);

    void setGravity(int i, int i2, int i3);

    void setMargin(float f, float f2);

    void setText(int i);

    void setText(CharSequence charSequence);

    void setView(View view);

    void show();

    /* renamed from: com.hjq.toast.config.IToast$-CC  reason: invalid class name */
    public final /* synthetic */ class CC {
        public static TextView $default$findMessageView(IToast _this, View view) {
            if (view instanceof TextView) {
                if (view.getId() == -1) {
                    view.setId(16908299);
                } else if (view.getId() != 16908299) {
                    throw new IllegalArgumentException("You must set the ID value of TextView to android.R.id.message");
                }
                return (TextView) view;
            } else if (view.findViewById(16908299) instanceof TextView) {
                return (TextView) view.findViewById(16908299);
            } else {
                throw new IllegalArgumentException("You must include a TextView with an ID value of android.R.id.message");
            }
        }
    }
}

File: decompiled/sources/com/hjq/toast/config/IToast.kt
package com.hjq.toast.config

import android.view.View
import android.widget.TextView

interface IToast {
    fun cancel()

    fun findMessageView(view: View): TextView {
        if (view is TextView) {
            if (view.id == View.NO_ID) {
                view.id = android.R.id.message
            } else if (view.id != android.R.id.message) {
                throw IllegalArgumentException("You must set the ID value of TextView to android.R.id.message")
            }
            return view
        }
        val messageView = view.findViewById<View>(android.R.id.message)
        if (messageView is TextView) {
            return messageView
        }
        throw IllegalArgumentException("You must include a TextView with an ID value of android.R.id.message")
    }

    fun getDuration(): Int

    fun getGravity(): Int

    fun getHorizontalMargin(): Float

    fun getVerticalMargin(): Float

    fun getView(): View?

    fun getXOffset(): Int

    fun getYOffset(): Int

    fun setDuration(duration: Int)

    fun setGravity(gravity: Int, xOffset: Int, yOffset: Int)

    fun setMargin(horizontalMargin: Float, verticalMargin: Float)

    fun setText(textId: Int)

    fun setText(text: CharSequence)

    fun setView(view: View?)

    fun show()
}
